classes/identity/Identity.inc.php

```diff
@@ -269,6 +275,30 @@ function getBiography($locale) {
 	function setBiography($biography, $locale) {
 		$this->setData('biography', $biography, $locale);
 	}
+	 
+	/**
+         * Get the localized firstName for this author
```

Comment:
Suggest using "for this identity", not "for this author". This is the parent class of both Author and PKPUser. (Also below)